core/digital-twin-lib/src/interfaces/DigitalTwinTemplate.ts

```diff
@@ -0,0 +1,20 @@
+import { DBCPDescriptionInterface } from '../DAppContract';
+import { ContainerPlugin } from '@evan.network/api-blockchain-core';
+
+export interface DigitalTwinTemplateInterface {
```

Comment:
You could also use the `DigitalTwinTemplate` interface from blockchain-core. https://github.com/evannetwork/api-blockchain-core/blob/develop/src/contracts/digital-twin/digital-twin.ts#L55

Reply:
ah nice I was looking for this, but I didn't found it. Probably dev wasn't up to date.

Reply:
Ahh yes, was merged only a hour ago...^^